### PR TITLE
update quantcastBidAdapter to pass quantcast fpa in the bid request

### PR DESCRIPTION
**Pull request by mckurt**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
This PR updates Quantcast bidding adapter to read the Quantcast first party cookie and include it in the bid request. Please contact asig@quantcast.com for any issues.

### CODE REVIEW

File: modules/lunamediahbBidAdapter.js [hunk 1]
```diff
@@ -70,7 +70,6 @@ export const spec = {
         schain: bid.schain || {},
       };
       const mediaType = bid.mediaTypes
-
```
Comment (mckurt) [inline modules/lunamediahbBidAdapter.js:73]: Build was complaining about this line, so I removed it:

"/home/circleci/Prebid.js/modules/lunamediahbBidAdapter.js
  73:1  error  Trailing spaces not allowed  no-trailing-spaces"

**Review by robertrmartinez — APPROVED** [review]
looks good

**Comment by mckurt** [comment]
@robertrmartinez Thanks for the review. When can we merge this PR?

File: modules/quantcastBidAdapter.js [hunk 2]
```diff
@@ -18,6 +19,9 @@ export const QUANTCAST_TEST_PUBLISHER = 'test-publisher';
 export const QUANTCAST_TTL = 4;
 export const QUANTCAST_PROTOCOL = 'https';
 export const QUANTCAST_PORT = '8443';
+export const QUANTCAST_FPA = '__qca';
+
+export const storage = getStorageManager();
```
Comment (robertrmartinez) [inline modules/quantcastBidAdapter.js:24]: Does Quantcast have a registered GVL id?

Passing the following will allow storage access when GDPR Enforcement is enabled.
https://github.com/prebid/Prebid.js/blob/master/modules/appnexusBidAdapter.js#L61

Comment (mckurt) [reply modules/quantcastBidAdapter.js]: We do. I updated the code accordingly. Thanks for the catch.

**Review by robertrmartinez — APPROVED** [review]
Cool 👍

**Pull request merged**